new_builder_evaluation/app/api/endpoints/aimodels.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) MLCommons and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from fastapi import APIRouter
+
+from app.domain.aimodels import AIModels
```

Comment:
I like the implementation, but I'm not sure about the class name. AIModels seems to broad.

Reply:
Any suggestions? I faced the same problem when I tried to set a name